src/vs/base/parts/quickinput/common/quickInput.ts

```diff
@@ -289,6 +289,8 @@ export interface IQuickPick<T extends IQuickPickItem> extends IQuickInput {
 
 	autoFocusOnList: boolean;
 
+	resetScrollPosition: boolean;
```

Comment:
For the API, I would turn this around because default values are never `true`

Reply:
flipped it around and called it `keepScrollPosition: boolean`